public/css/user/home.css

```diff
@@ -177,7 +177,7 @@ section {
   font-size: 70px;
 }
 
-.footer--contactUs-contactUs {
+.footer-inputs {
```

Comment:
@aminalakhsham this looks good! This is exactly what I had in mind with my comments earlier today.